components/entitlement/org.wso2.carbon.identity.entitlement.ui/src/main/resources/web/entitlement/my-pdp.jsp

```diff
@@ -97,7 +97,7 @@
 
     } catch (Exception e) {
     	String message = resourceBundle.getString("error.while.loading.policy")+ " " + e.getMessage();
-        CarbonUIMessage.sendCarbonUIMessage(message, CarbonUIMessage.ERROR, request, e);
+        CarbonUIMessage.sendCarbonUIMessage(message, CarbonUIMessage.ERROR, request);
```

Comment:
Until kernel is released with the fix wso2/carbon-kernel#1652, it is not secure to send exceptions which may contain user inputs, to the CarbonUIMessage.